devicon.json

```
@@ -3310,6 +3310,33 @@
         "color": "#2D333A",
         "aliases": []
     },
+    {
+        "name": "socketio",
+        "tags": [
+            "library",
+            "networking",
+            "websockets"
+        ],
+        "versions": {
+            "svg": [
+                "original",
+                "original-wordmark",
+                "plain",
+                "plain-wordmark"
+            ],
+            "font": [
+                "plain",
+                "plain-wordmark"
+            ]
+        },
+        "color": "#010101",
+        "aliases": [
+            {
+                "base": "original",
+                "alias": "plain"
+            }
```

If you'd still like to use `aliases`, please do this for all base-alias pairs. `original` -> `plain` will only create an alias for the `original`, not the `original-wordmark`.

```suggestion
            },
            {
              "base": "original-wordmark",
              "alias": "plain"
            }
```